### PR TITLE
ignore VFs defined 

### PR DESCRIPTION
**Pull request by SharonBX**
currently os-net-config fail to rerun in case config.json not includes sriov VF definitions (or sriov been defined on the host without nmstate ) the fail reason is the desire VFs hard coded to zero is not as current.
this pr will replace the zero with current defined VFs to avoid the fail verification.

### CODE REVIEW

File: os_net_config/impl_nmstate.py
Comment (karthiksundaravel) [inline os_net_config/impl_nmstate.py:46]: Don't import sriov_config here. The numvfs could be obtained from the iface_state() itself.

**Review by karthiksundaravel — CHANGES_REQUESTED** [review]
Its tricky one and I'll have to check how this patch fits the other Openstack use cases.

**Review by dsneddon — APPROVED** [review]

**Review by dsneddon — APPROVED** [review]

**Review by dsneddon — APPROVED** [review]
This looks good to me, but I will wait for a review from Karthik before merging. Thanks!

**Review by karthiksundaravel — APPROVED** [review]

**Pull request merged**
